ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -13,8 +13,8 @@
 
 @st.composite
 def statistical_dtype_values(draw, *, function):
-    large_abs_safety_factor = 72
-    small_abs_safety_factor = 72
+    large_abs_safety_factor = 24
+    small_abs_safety_factor = 24
     if function in ["mean", "median", "std", "var"]:
```

Comment:
This change is only to be done if it fails reverting the `safety factor = 2`:
 `if function in ["mean", "median", "std", "var", "nanquantile"]:`
 This condition will set the safety factor to 24.